### PR TITLE
Close TablePrx services

### PR DESCRIPTION
**Pull request by joshmoore**
This solves issues with the automated tests since files
are not left open leading to `None` return values from
`openTable`.

### CODE REVIEW

File: notebooks/Figure_1_Sampling_of_Phenotypes.ipynb [hunk 1]
```diff
@@ -377,19 +377,23 @@
...
+    "            column_indices.append(column_names.index(column_name))\n",
     "\n",
-    "    black_list = []\n",
-    "    column_indices = []\n",
-    "    for column_name in column_names:\n",
-    "        if column_name in black_list:\n",
-    "            continue\n",
-    "        column_indices.append(column_names.index(column_name))\n",
+    "        table_data = table.slice(column_indices, None)\n",
+    "    finally:\n",
```
Comment (jburel) [inline notebooks/Figure_1_Sampling_of_Phenotypes.ipynb]: I missed that one last round

**Comment by jburel** [comment]
I missed the table last round.
Changes make sense

**Comment by joshmoore** [comment]
@manics : is it feasible to deploy this in prod47?

**Comment by manics** [comment]
Can do. You'll need to update the jupyter-docker image (or create a separate one), update the idr/deployment image spec, and redeploy jupyterhub

**Comment by joshmoore** [comment]
Decided today to focusing on getting these changes onto idr-analysis rather than prod47

**Comment by jburel** [comment]
Note that this PR has broken PCAanalysis
The table is closed before the content is read

**Pull request merged**